Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -213,7 +311,12 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
     setTriggerModalVisibility(true);
   };
 
-  const onTriggerCreationSubmit = (dialog: DialogInfo, intent?: LuIntentSection) => {
+  const onTriggerCreationSubmit = async (
```

Comment:
Can we lift up a trigger creation action to shell level, for example: 
a `createTrigger` method which combine `createLg, createLu, updateDialog`

Reply:
Weitian may have some progress on it, that's exactly what we talked about this morning

Reply:
Yeah, we still can package those progress into one higher-level action, let shell action store control the side effect.  If other component have the same business logic, it just need call a packaged action.